### DISCUSSION TITLE
Searx image search problem

### DISCUSSION THREAD

**Discussion by AkiraOtosaka** [searx/searx#2793] [Public Searx instances: report a bug]
**Version of Searx, commit number if you are using on master branch and stipulate if you forked Searx**

If you are using searx-docker then look at the bottom of the Searx page
and check for the version after "Powered by searx"
Powered by searx - 0.18.0-328-a48ec0b4 

**How did you install Searx?**
I use public instances

**What happened?**
When i search for images, after i scroll down a little bit, it doesn't load the images, some othertimes, shows the same image bcz of multiple search engines

**How To Reproduce**
Idk what "Reproduce" means

**Expected behavior**
I want the image search to be normal, like what any other search engines do

**Screenshots & Logs**

https://tgfilestreamer.herokuapp.com/19873763380912140/2021-05-01_11:56:25.jpg

**Additional context**

Nothing else really...

**Comment by allendema** [comment]
It could be that the instance is misconfigured / somehow blocking engines.

Can you try it with another instance?

**Comment by kvch** [comment]
By reproducing we mean that you could share the query you had run that gave you this incomplete output.
